.github/workflows/ci.yml

```diff
@@ -136,6 +136,33 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 10
+      zookeeper:
+        image: bitnami/zookeeper
+        ports:
+          - 2181:2181
+        env:
+          ALLOW_ANONYMOUS_LOGIN: yes
+        options: >-
+          --health-cmd "echo mntr | nc -w 2 -q 2 localhost 2181"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      kafka:
+        image: bitnami/kafka
```

Comment:
What does the use of 3rd party Bitnami images for Kafka and Zookeeper (over the official Apache ones) provide us with?

Reply:
I'm not sure actually, I was following some instructions/tutorial type stuff for getting this running on github actions and this is what was used there (although i don't have that link anymore, i closed the tab a while ago). I could try changing it if you think it'd be good it try, but this is what I saw being used to github actions specifically, so perhaps there is a reason, i'm just not aware of it.